my-react-app/src/App.js

```diff
@@ -1,24 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import LoginPage from "./components/LoginPage";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Switch>
+        <Route path="/" component={LoginPage} />
+      </Switch>
```

Comment:
Main route should point to the landing page instead of the login page. I will edit the router for all routes, so don't worry about this one.